music21/musicxml/m21ToXml.py

```diff
@@ -6062,6 +6062,16 @@ def staffLayoutToXmlStaffDetails(self, staffLayout):
             mxStaffLines.text = str(staffLayout.staffLines)
 
         # TODO: staff-tuning
```

Comment:
are there further scenarios involving `staff-tuning` that won't be handled? let's update or remove the comment.